docs/examples/Example_data_fusion_from_sensors.py

```diff
@@ -0,0 +1,506 @@
+    position_mapping=(0, 2),
+    noise_covar=radar_noise,
+    clutter_model=clutter_model,
+    max_range=3000)
+
+# Import the platform to place the sensors on
+from stonesoup.platform.base import FixedPlatform
+from stonesoup.platform.base import MovingPlatform
+
+# Instantiate the first sensor platform and add the sensor
+sensor1_platform = FixedPlatform(
+    states=GaussianState([10, 0, 5, 0],
+                         np.diag([1, 0, 1, 0])),
+    position_mapping=(0, 2),
+    sensors=[radar1])
+
+sensor2_platform = MovingPlatform(
+    states=GaussianState([120, 0, -50, 1.5],
+                         np.diag([1, 0, 5, 1])),
+    position_mapping=(0, 2),
+    velocity_mapping=(0, 2),
+    transition_model=gnd_transition_model,
+    sensors=[radar2])
+
+
+# Load the platform detection simulator - Let's use a simulator for each track
+# Instantiate the simulators
+from stonesoup.simulator.platform import PlatformDetectionSimulator
+
+radar_simulator1 = PlatformDetectionSimulator(
+    groundtruth=ground_truth_simulator,
+    platforms=[sensor1_platform])
+
+radar_simulator2 = PlatformDetectionSimulator(
+    groundtruth=ground_truth_simulator,
+    platforms=[sensor2_platform])
+
+
+# %%
+# 2) Define the various filter components and build the trackers
+# --------------------------------------------------------------
+# We have presented the scenario with two separate moving targets
+# and two sensors collecting the measurements. Now, we focus on
+# building the various tracker components: we use a
+# :class:`~.DistanceHypothesiser`
+# hypothesiser using :class:`~.Mahalanobis` distance measure
+# to assign detections to tracks.
+# We consider an Unscented Kalman filter (UKF), an Extended Kalman filter
+# (EKF) and a Particle filter (PF) using the same components specifications.
+# For the deleter we consider an :class:`~.UpdateTimeDeleter`.
+#
+
+# %%
+# Stone Soup imports
+# ^^^^^^^^^^^^^^^^^^
+
+# We use a Distance hypothesiser
+from stonesoup.hypothesiser.distance import DistanceHypothesiser
+from stonesoup.measures import Mahalanobis
+
+# Use a GNN 2D assignment
+from stonesoup.dataassociator.neighbour import GNNWith2DAssignment
+from stonesoup.deleter.time import  UpdateTimeDeleter
+from stonesoup.initiator.simple import MultiMeasurementInitiator
+
+# Load the UKF components
+from stonesoup.updater.kalman import UnscentedKalmanUpdater
+from stonesoup.predictor.kalman import UnscentedKalmanPredictor
+
+# Load the EKF components
+from stonesoup.updater.kalman import ExtendedKalmanUpdater
+from stonesoup.predictor.kalman import ExtendedKalmanPredictor
+
+
+# prepare the Particle filter components
+from stonesoup.predictor.particle import ParticlePredictor
+from stonesoup.updater.particle import ParticleUpdater
+from stonesoup.resampler.particle import ESSResampler
+
+# prepare the Particle initiator
+from stonesoup.initiator.simple import SimpleMeasurementInitiator, GaussianParticleInitiator
+
+# %%
+# Design the trackers components
```

Comment:
This is quite a long section of code for an example, could it be reorganised a bit? Perhaps split into different cells for each tracker type to make it a bit more readable.